__tests__/api.test.js

```diff
@@ -0,0 +1,38 @@
+const app = require("../app.js");
+const request = require("supertest");
+const db = require("../db/connection.js");
+const seed = require("../db/seeds/seed.js");
+const data = require("../db/data/test-data");
+
+beforeEach(() => seed(data));
+afterAll(() => db.end());
+
+describe("/api", () => {
+  it("GET - 200: Responds with message 'connection ok'", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then((res) => {
+        expect(res.body).toEqual({
+          message: "connection ok",
+        });
+      });
+  });
+});
+
+describe("/api/topics", () => {
+  test("GET:200 sends an array of topics to the client", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then((response) => {
+        expect(response.body.topics).toEqual(expect.any(Array));
+        response.body.topics.forEach((topic) => {
+          expect(topic).toMatchObject({
+            slug: expect.any(String),
+            description: expect.any(String),
+          });
```

Comment:
You need to check the length of the array is a non-zero value - an empty array would pass this test.

Reply:
Done